Havit.Blazor.Components.Web.Bootstrap.Documentation/Pages/Components/HxAutosuggestDoc/HxAutosuggest_Documentation.razor

```diff
@@ -12,5 +12,8 @@
         <ComponentApiDocCssVariable Name="--hx-autosuggest-input-close-icon-opacity" Default=".25">
             Opacity of the close icon.
         </ComponentApiDocCssVariable>
+        <ComponentApiDocCssVariable Name="--hx-autosuggest-item-hightlighted-background-color" Default="var(--bs-gray-200)">
+            When <code>HxAutosuggest.HighlightFirstSuggestion</code> is <code>true</code>, then the first item receives this background color.
```

Comment:
Is it just the first item, or does the CSS rule apply to all other items focused?

Reply:
The class in "HxAutosuggestItems.razor.css" has been renamed to better convey the meaning.